examples/arc_ball_camera/README.md

```diff
@@ -0,0 +1,2 @@
+# Fly Camera Example
```

Comment:
Rather than a dedicated readme it would actually be more important to add it to the overview.